addons/casings/XEH_postInit.sqf

```diff
@@ -0,0 +1,7 @@
+#include "script_component.hpp"
+
+if (!hasInterface || !(GVAR(enabled))) exitWith {};
```

Comment:
```suggestion
if (!hasInterface || {!GVAR(enabled)}) exitWith {};
```
Eliminates the need to unnecessarily check both, and pretty sure the parentheses ain't needed.

Reply:
This is slower on average for any mission with more than 1 player connected, regardless whether casings are enabled or not.

Reply:
I do support axing the parentheses though.

Reply:
> This is slower on average for any mission with more than 1 player connected, regardless whether casings are enabled or not.

Huh. Complete contrast to what Biwiki said.

Reply:
>BI Wiki

LOL
Where do you think you are?

Reply:
it's 0.0007ms vs 0.0008ms here guys for something that runs once. Binned parentheses .

Reply:
> > BI Wiki
> 
> LOL Where do you think you are?

Possibly remembered the spelling wrong but the point gets across.